src/setup/genesis/index.js

```diff
@@ -20,7 +20,7 @@ export class Genesis {
         this.config = config;
 
         this.repositoryName = this.name;
-        this.repositoryBranch = "mardizzone/node-upgrade";
+        this.repositoryBranch = "arpit/pos-655";
         this.repositoryUrl =
```

Comment:
Hey @temaniarpit27 
I'd suggest we merge `arpit/pos-655` genesis branch into `mardizzone/node-upgrade` so that we can leverage node `v16.17.1` for your changes too. What do you think?

Once we do that, I'd revert this branch to `"mardizzone/node-upgrade"` and we can merge this one too.

Also, added this one https://polygon.atlassian.net/browse/POS-984 for future implementation.

Reply:
`arpit/pos-655` already contains the changes from `mardizzone/node-upgrade`
i have made this PR on top of your PR

Reply:
Ok, thanks